src/promptflow/promptflow/_sdk/_service/apis/experiment.py

```diff
@@ -0,0 +1,31 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from flask import jsonify, request
+
+from promptflow._sdk._constants import get_list_view_type
+from promptflow._sdk._service import Namespace, Resource
+from promptflow._sdk._service.utils.utils import get_client_from_request
+
+
+api = Namespace("Experiments", description="Experiments Management")
+
+# Response model of experiment operation
+dict_field = api.schema_model("ExperimentDict", {"additionalProperties": True, "type": "object"})
```

Comment:
will you add those API required for chat ui in this PR?

Reply:
doesn't get it, will offline sync